tests/api2/test_425_smb_protocol.py

```diff
@@ -582,29 +634,37 @@ def test_151_set_xattr_via_ssh(request, xat):
 
 @pytest.mark.dependency(name="XATTR_CHECK_SMB_READ")
 @pytest.mark.parametrize('xat', AFPXattr.keys())
-def test_152_check_xattr_via_smb(request, xat):
+def test_152_check_xattr_via_smb(request, mount_share, xat):
     """
     Read xattr that was written via SSH and verify that
     data is same when viewed over SMB.
     """
     depends(request, ["SSH_XATTR_SET"])
     afptestfile = f'afp_xattr_testfile:{AFPXattr[xat]["smbname"]}'
-    bytes = AFPXattr[xat]["smb_bytes"] if xat == "org.netatalk.Metadata" else AFPXattr[xat]["bytes"]
+    bytes_to_read = AFPXattr[xat]["smb_bytes"] if xat == "org.netatalk.Metadata" else AFPXattr[xat]["bytes"]
     c = SMB()
     c.connect(host=ip, share=SMB_NAME, username=SMB_USER, password=SMB_PWD, smb1=False)
     fd = c.create_file(afptestfile, "w")
-    xat_bytes = c.read(fd, 0, len(bytes) + 1)
+    xat_bytes = c.read(fd, 0, len(bytes_to_read) + 1)
     c.close(fd)
     c.disconnect()
 
     err = {
         "name": xat,
-        "b64data": b64encode(bytes)
+        "b64data": b64encode(bytes_to_read)
```

Comment:
Shouldn't this be `xat_bytes` rather than `bytes_to_read`?

(`bytes_to_read` is the well-known input)